### PR TITLE
docs: add platform v10 post

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: blog/2022-07-27-app-platform-v10.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+slug: 2022/07/app-platform-v10
+title: App-platform v10
+author: ismay
```
Comment (Mohammer5) [inline blog/2022-07-27-app-platform-v10.md:4]:
```suggestion
author: Ismay
``` 

:laughing:

File: blog/2022-07-27-app-platform-v10.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+tags: [dhis2, app-platform]
+---
+
+We have just released a new major version of the app-platform: version 10. This new version should be relatively easy to update to for everyone. In this post we'll walk you through the most important change for this new version.
```
Comment (Mohammer5) [inline blog/2022-07-27-app-platform-v10.md:8]:
```suggestion
We have just released a new major version of the app-platform: version 10. It should be relatively easy to update to for everyone. In this post we'll walk you through the breaking change of this release.
```

4 times "version" in two sentences sounds a bit awkward to me

Comment (amcgee) [inline blog/2022-07-27-app-platform-v10.md:8]:
```suggestion
We have just released a new major version of the app-platform: version 10. Upgrading to this new version should be relatively easy for all apps currently using version 9 of `cli-app-scripts`. In this post we'll walk you through the most important change for this new version.
```

File: blog/2022-07-27-app-platform-v10.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Node support
+
+The one breaking change for this release is that we've dropped support for node 12. This means that the app-platform libraries now only support the current LTS versions of node, which are node 14 and up. Node 12 is no longer under active development and we were seeing some of our dependencies dropping support for it, so we felt it was time we did the same.
```
Comment (Mohammer5) [inline blog/2022-07-27-app-platform-v10.md:12]:
```suggestion
The one breaking change for this release is that we've dropped support for node 12. This means that now only the current LTS versions of nodejs will be supported by the app-platform libraries, which is node 14 and up. Node 12 is no longer under active development and we were seeing some of our dependencies dropping support for it, so we felt it was time we did the same.
```

Comment (Mohammer5) [reply blog/2022-07-27-app-platform-v10.md:12]: I have no idea if what I suggested is correct at all, just suggested what sounds "more correct" to me, please refer to a native speaker or simply ignore my suggestion

**Review by Mohammer5 — APPROVED** [review]

File: blog/2022-07-27-app-platform-v10.md (hunk 3 above)
Comment (amcgee) [inline blog/2022-07-27-app-platform-v10.md]:
```suggestion
There is only one breaking change in this major release.  We have dropped support for node 12, meaning that the app-platform libraries now only support node 14 and node 16, the current LTS versions of node. Node 12 is no longer under active development and some of our dependencies are dropping support for it, so we felt it was time we did the same.
```

File: blog/2022-07-27-app-platform-v10.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+It should be an easy upgrade for everyone, all you'll have to do is update the version of node you're using locally or on CI. If you encounter any issues let us know! For a detailed overview of what’s changed you can view the [changelog](https://github.com/dhis2/app-platform/blob/master/CHANGELOG.md#1000-2022-07-26)
```
Comment (amcgee) [inline blog/2022-07-27-app-platform-v10.md:14]:
```suggestion
It should be an easy upgrade for everyone, all you'll have to do is update the version of node you're using locally or in your continuous integration (CI) pipelines. If you encounter any issues let us know! For a detailed overview of all the changes in this platform release you can view the [changelog](https://github.com/dhis2/app-platform/blob/master/CHANGELOG.md#1000-2022-07-26)
```

**Review by amcgee — COMMENTED** [review]
Minor grammar / clarity suggestions.  @ismay I think you also need to add yourself to the authors.yml file

**Comment by amcgee** [comment]
https://github.com/dhis2/developer-portal/blob/main/blog/authors.yml

**Review by amcgee — APPROVED** [review]

**Pull request merged**